examples/mnist/README.md

```diff
@@ -57,3 +57,23 @@ Run the example:
 python mnist_with_visdom.py
 ```
 
+## Distributed training
+
+#### Usage:
```

Comment:
@snowyday sorry for the delay. I think it would be better to give the cases like in imagenet example:
- Single node, multiple GPUs:
   - command
- Multiple nodes with multiple GPUs
  - another command

Reply:
@vfdev-5 Thank you for considering the examples. I added two cases 20c46778c16927adb13cf5e53a14cb6f5688fdbb.

Reply:
@snowyday thanks! I'll try to execute this today and give you my feedback